client/src/components/Explorer/ExplorerItem.tsx

```diff
@@ -26,6 +26,7 @@ const ExplorerItem: React.FunctionComponent<ExplorerItemProps> = ({ item, onClic
   const { id, admin_display_title: title, meta } = item;
   const hasChildren = meta.children.count > 0;
   const isPublished = meta.status.live && !meta.status.has_unpublished_changes;
+  const localeName = meta.parent?.id === 1 && meta.locale && (LOCALE_NAMES.get(meta.locale) || meta.locale);
```

Comment:
I don't know this syntax, `?.`. But I guess it is to check if `parent` exists and if so, get the `id`?

Reply:
Yep, that's right. ``meta.parent?.id`` is equivilant to ``meta.parent && meta.parent.id``